src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    private String name;
+    private int quantity;
+
+    public Fruit(String name) {
+        this.name = name;
+        this.quantity = 0;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public void addQuantity(int quantity) {
+        this.quantity += quantity;
+    }
+
+    public void subtractQuantity(int quantity) {
+        this.quantity -= quantity;
+    }
+}
```

Comment:
Consider following the encapsulation principle more strictly. Since the 'quantity' field is being managed through 'addQuantity' and 'subtractQuantity' methods, there is no need for public setters and getters for 'quantity'. You can remove the 'setQuantity' and 'getQuantity' methods to ensure that 'quantity' is only modified through the provided methods, which encapsulate the logic for quantity changes.